src/main/java/com/powsybl/caseserver/CaseService.java

```diff
@@ -346,7 +346,7 @@ public void setComputationManager(ComputationManager computationManager) {
 
     static void validateCaseName(String caseName) {
         Objects.requireNonNull(caseName);
-        if (!caseName.matches("[\\w\\-]+(\\.[\\w]+)*+")) {
+        if (!caseName.matches("[\\w(\\)\\-]+(\\.[\\w]+)*+")) {
```

Comment:
Why not :  if (!caseName.matches("[\\w\\(\\)\\-]+(\\.[\\w]+)*+")) {

And parentheses in the extension part doesn't function : case-server cannot find an importer in that case)